### ISSUE TITLE
44 Nomad 6villager 250pop, this gamesetting carshed at time 1:00

### ISSUE THREAD

**Issue by cliffture** [Woprok/AOE4-AdvancedGameSettings#31]
Hi there, I met this issue every time I tried. So sad, looking forward to solve it.
“**Synchronization error detected**“，and the game crashed.

**Comment by Woprok** [comment]
@cliffture Can you give little more info ? I am aware that my mod for some unknown reason desyncs, so I am trying to pinpoint exact problem. Does it happen with scattered randomly around the map or with villagers grouped at the start location without TC ? Or with both ? Any other settings that was active ?

**Comment by cliffture** [comment]
It works well with AI only. But it crashed when I play with other players. I choosed Mega Random Map and this mod,  here are my Settings.
![1](https://user-images.githubusercontent.com/26118456/168090906-f418e9d4-c173-4908-b8c2-9b04f3d0df65.png)
![2](https://user-images.githubusercontent.com/26118456/168090940-02b42490-693b-46be-bbf8-ca91150bd384.png)
![3](https://user-images.githubusercontent.com/26118456/168090967-102cce67-861c-45cc-b7f6-1b0c60d1534b.png)
![4](https://user-images.githubusercontent.com/26118456/168091000-c5deec19-2864-42d4-a490-ccf24557d6ff.png)
It works fine at first, I can find my 6 villager scattered randomly around the map but at Time 1:00 it crashed. I tried twice, same result.
